File: PPA/DailyFlash/August/07August/Program1.c


#include<stdio.h>

void main() {
	int num1,num2;
	printf("Enter two numbers: ");
	scanf("%d%d",&num1,&num2);

	int store = num1;
	int num_2=num2;
	do {
		
		store =store +num1;
		num2--;
	}while(num2>1);
	printf("Multiplication of %d and %d is %d.\n",num1,num_2,store);
}

File: Java/Assignments/Assignment/IOAssignment2-1/Program1.java

import java.io.*;

class Operation {

	public static void main(String[] args) throws IOException {

		InputStreamReader isr = new InputStreamReader(System.in);
		BufferedReader br = new BufferedReader(isr);

		int num1, num2, choice;

		System.out.print("Enter num1 and num2: ");
		num1=Integer.parseInt(br.readLine());
		num2=Integer.parseInt(br.readLine());
		
		System.out.println("\nEnter your choice.\n1)Addition\n2)Subtraction\n3)Division\n4)Multiplication\n");
		choice=Integer.parseInt(br.readLine());
		switch(choice) {

			case 1:
				System.out.println("Addition = "+(num1+num2));
				break;
			case 2:
				System.out.println("Subtraction = "+(num1-num2));
				break;
			case 3:
				System.out.println("Division = "+(num1*num2));
				break;
			case 4:
				System.out.println("Multiplication = "+(num1*num2));
				break;
			default:
				System.out.println("U entered a wrong choice!");
				break;
		}
	}
}
